modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart

```diff
@@ -38,6 +38,10 @@ class Emails {
         onValidationFailure,
     void Function(Object e)? onError,
   }) async {
+    if (AuthConfig.current.validatePasswordHashCallback != null) {
```

Comment:
Discussion/Question: How should we handle `onError` and `onValidationFailure` for the custom implementation?

Reply:
That makes sense 👍